README.md

```diff
@@ -25,6 +25,7 @@ Read more about developing Hitchwiki [from the wiki](https://github.com/Hitchwik
 * Make sure you have [`git`](http://git-scm.com/) installed on your system. (`git --version`)
 
 ### Install
+
```

Comment:
It would probably be most clear to separate these to two different files: `INSTALL.md` and `INSTALL-VAGRANT.md` — WDYT?

See TR docs for comparison:
https://github.com/Trustroots/trustroots/blob/master/README.md#installing
https://github.com/Trustroots/trustroots/blob/master/INSTALL.md
https://github.com/Trustroots/trustroots/blob/master/INSTALL-DOCKER.md